setup.py

```diff
@@ -2,6 +2,7 @@
     from setuptools import setup
 except ImportError:
     from distutils.core import setup
+from Cython.Build import cythonize
```

Comment:
This should be in a `try/except ImportError`. On error, tell the user to install cython (`pip install cython`) as it is required for the `ext_modules` line. And, perhaps on line 27 'Cython' can be removed.